simplemind/providers/amazon.py

```
@@ -0,0 +1,87 @@
+        """The AnthropicBedrock client."""
+        if not self.profile_name:
+            raise ValueError("Profile name is not provided")
+
+        return anthropic.AnthropicBedrock(aws_profile=self.profile_name)
+
+    @property
+    def structured_client(self):
+        """A client patched with Instructor."""
+        return instructor.from_anthropic(self.client)
+
+    def send_conversation(self, conversation: "Conversation", **kwargs):
```

Ruff

Undefined name `Conversation`

(F821)